src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.tsx

```diff
@@ -7,7 +7,7 @@ import s from "./TwoParagraphBlock.module.css";
 /* Main component */
 type TwoParagraphBlockProps = {
   title: string;
-  paragraph1: React.ReactNode;
+  paragraph1: string;
```

Comment:
Is there a reason you needed to change this? `ReactNode` is a type alias that includes `string` as one of its valid subtypes, so you should be able to pass a string into a ReactNode prop. I think we'd prefer the `ReactNode` here for symmetry with `paragraph2`, where we're currently passed in something more general than a string, JSX code.